--- ejabberdctl.orig	2018-10-06 12:25:28.773110703 +0200
+++ ejabberdctl	2018-10-06 12:26:08.825514910 +0200
@@ -56,6 +56,7 @@
 
 # define ejabberd variables if not already defined from the command line
 : "${ETC_DIR:="/etc/ejabberd"}"
+: "${ERL_INETRC:="/etc/inetrc"}"
 : "${LOGS_DIR:="/var/log/ejabberd"}"
 : "${SPOOL_DIR:="/var/lib/ejabberd"}"
 : "${EJABBERD_CONFIG_PATH:="$ETC_DIR/ejabberd.yml"}"
@@ -79,7 +80,6 @@
 fi
 ERL_LIBS=/usr/lib/x86_64-linux-gnu
 ERL_CRASH_DUMP="$LOGS_DIR"/erl_crash_$(date "+%Y%m%d-%H%M%S").dump
-ERL_INETRC="$ETC_DIR"/inetrc
 
 # define ejabberd parameters
 EJABBERD_OPTS="$EJABBERD_OPTS\
